dartagnan/src/main/java/com/dat3m/dartagnan/witness/graphviz/ExecutionGraphVisualizer.java

```diff
@@ -311,10 +305,11 @@ private String eventToNode(EventModel e) {
         }
         final String callStack = makeContextString(
             synContext.getContextInfo(e.getEvent()).getContextOfType(CallContext.class), " -> \\n");
-        final String nodeString = String.format("%s:T%s/E%s\\n%s%s\n%s",
+        final String nodeString = String.format("%s:T%s/E%s%s\\n%s%s\n%s",
```

Comment:
I would rather put the scope hierarchy next to the thread and move the event id to the next line (together with the line of code where it comes from)

Reply:
Done.